cartridge-cli.lua

```diff
@@ -2670,34 +2873,48 @@ local function app_pack(args)
         end
     end
 
-    if args.type == 'rpm' then
-        pack_rpm({
-            unit_template = unit_template,
-            instantiated_unit_template = instantiated_unit_template
-        })
-    elseif args.type == 'deb' then
-        pack_deb({
+    local opts
+    if array_contains({distribution_types.RPM, distribution_types.DEB}, args.type) then
+        opts = {
             unit_template = unit_template,
             instantiated_unit_template = instantiated_unit_template
-        })
-    elseif args.type == 'tgz' then
-        pack_tgz()
-    elseif args.type == 'rock' then
-        pack_rock()
-    elseif args.type == 'docker' then
-        pack_docker({
+        }
+    elseif args.type == distribution_types.DOCKER then
+        opts = {
             tag = args.tag,
             from = args.from,
             download_token = args.download_token,
             docker_build_args = args.docker_build_args,
-        })
-    else
-        die("Unknown package type: %s", args.type)
+        }
+    end
+
+    local pack_handler = pack_handlers[args.type]
+    if pack_handler == nil then
+        local handler_err = string.format("Pack handler for %s distribution type not found", args.type)
```

Comment:
Do you really ask me to report a bug in case if I write `cartridge pack rar`?

Reply:
In your case you will get an error from `app_pack_parse` function, but if there is no pack handler for one of allowed types, it's internal error.

Reply:
ok